From fcc8aa45c6dae523d2a9748ab34c64aa9fe424a8 Mon Sep 17 00:00:00 2001
From: Bruce Korb <bkorb@gnu.org>
Date: Sun, 14 Sep 2014 12:51:20 -0700
Subject: [PATCH] licensing & DASH shell tweak

* agen5/test/format.test: export AUTOGEN_DNE_DATE explicitly.
  DASH does not export when the assignment is on the same line
  as a shell function invocation.
* autoopts/tpl/*.pl: document the licensing of the perl scripts.
---
 autoopts/tpl/Mdoc.pm      | 27 +++++++++++++++++++++++++++
 autoopts/tpl/man2mdoc.pl  | 27 ++++++++++++++++++++++++---
 autoopts/tpl/mdoc2man.pl  | 27 +++++++++++++++++++++++++--
 autoopts/tpl/mdoc2texi.pl | 27 +++++++++++++++++++++++++--
 4 files changed, 101 insertions(+), 7 deletions(-)
 mode change 100755 => 100644 autoopts/tpl/mdoc2texi.pl

diff --git a/autoopts/tpl/Mdoc.pm b/autoopts/tpl/Mdoc.pm
index 61fffd6..4b6b07c 100644
--- a/autoopts/tpl/Mdoc.pm
+++ b/autoopts/tpl/Mdoc.pm
@@ -1,3 +1,30 @@
+=begin comment
+
+## Mdoc.pm -- Perl functions for mdoc processing
+##
+## Author:	Oliver Kindernay (GSoC project for NTP.org)
+##		
+##
+##  This file is part of AutoOpts, a companion to AutoGen.
+##  AutoOpts is free software.
+##  AutoOpts is Copyright (C) 1992-2014 by Bruce Korb - all rights reserved
+##
+##  AutoOpts is available under any one of two licenses.  The license
+##  in use must be one of these two and the choice is under the control
+##  of the user of the license.
+##
+##   The GNU Lesser General Public License, version 3 or later
+##      See the files "COPYING.lgplv3" and "COPYING.gplv3"
+##
+##   The Modified Berkeley Software Distribution License
+##      See the file "COPYING.mbsd"
+##
+##  These files have the following sha256 sums:
+##
+##  8584710e9b04216a394078dc156b781d0b47e1729104d666658aecef8ee32e95  COPYING.gplv3
+##  4379e7444a0e2ce2b12dd6f5a52a27a4d02d39d247901d3285c88cf0d37f477b  COPYING.lgplv3
+##  13aa749a5b0a454917a944ed8fffc530b784f5ead522b1aacaf4ec8aa55a6239  COPYING.mbsd
+=end comment
 =head1 NAME
 
 Mdoc - perl module to parse Mdoc macros
diff --git a/autoopts/tpl/man2mdoc.pl b/autoopts/tpl/man2mdoc.pl
index 874a73b..b023629 100755
--- a/autoopts/tpl/man2mdoc.pl
+++ b/autoopts/tpl/man2mdoc.pl
@@ -1,8 +1,29 @@
 #!/usr/bin/perl
 
-#
-# written by Harlan Stenn
-#
+## man2mdoc.pl -- Convert man tags to mdoc tags
+##
+## Author:	Harlan Stenn <stenn@ntp.org>
+##		
+##
+##  This file is part of AutoOpts, a companion to AutoGen.
+##  AutoOpts is free software.
+##  AutoOpts is Copyright (C) 1992-2014 by Bruce Korb - all rights reserved
+##
+##  AutoOpts is available under any one of two licenses.  The license
+##  in use must be one of these two and the choice is under the control
+##  of the user of the license.
+##
+##   The GNU Lesser General Public License, version 3 or later
+##      See the files "COPYING.lgplv3" and "COPYING.gplv3"
+##
+##   The Modified Berkeley Software Distribution License
+##      See the file "COPYING.mbsd"
+##
+##  These files have the following sha256 sums:
+##
+##  8584710e9b04216a394078dc156b781d0b47e1729104d666658aecef8ee32e95  COPYING.gplv3
+##  4379e7444a0e2ce2b12dd6f5a52a27a4d02d39d247901d3285c88cf0d37f477b  COPYING.lgplv3
+##  13aa749a5b0a454917a944ed8fffc530b784f5ead522b1aacaf4ec8aa55a6239  COPYING.mbsd
 
 use strict;
 
diff --git a/autoopts/tpl/mdoc2man.pl b/autoopts/tpl/mdoc2man.pl
index d48d969..f511e60 100644
--- a/autoopts/tpl/mdoc2man.pl
+++ b/autoopts/tpl/mdoc2man.pl
@@ -1,12 +1,35 @@
 #!/usr/bin/perl
 
+## mdoc2man.pl -- Convert mdoc tags to man tags
+##
+## Author:	Harlan Stenn <stenn@ntp.org>
+##		
+##
+##  This file is part of AutoOpts, a companion to AutoGen.
+##  AutoOpts is free software.
+##  AutoOpts is Copyright (C) 1992-2014 by Bruce Korb - all rights reserved
+##
+##  AutoOpts is available under any one of two licenses.  The license
+##  in use must be one of these two and the choice is under the control
+##  of the user of the license.
+##
+##   The GNU Lesser General Public License, version 3 or later
+##      See the files "COPYING.lgplv3" and "COPYING.gplv3"
+##
+##   The Modified Berkeley Software Distribution License
+##      See the file "COPYING.mbsd"
+##
+##  These files have the following sha256 sums:
+##
+##  8584710e9b04216a394078dc156b781d0b47e1729104d666658aecef8ee32e95  COPYING.gplv3
+##  4379e7444a0e2ce2b12dd6f5a52a27a4d02d39d247901d3285c88cf0d37f477b  COPYING.lgplv3
+##  13aa749a5b0a454917a944ed8fffc530b784f5ead522b1aacaf4ec8aa55a6239  COPYING.mbsd
+
 ### ToDo
 # Properly implement -columns in the "my %lists" definition...
 #
 # .Xr requires at least 1 arg, the code here expects at least 2
 #
-# written by Harlan Stenn
-#
 ###
 
 package mdoc2man;
diff --git a/autoopts/tpl/mdoc2texi.pl b/autoopts/tpl/mdoc2texi.pl
old mode 100755
new mode 100644
index 432d71d..1395bbf
--- a/autoopts/tpl/mdoc2texi.pl
+++ b/autoopts/tpl/mdoc2texi.pl
@@ -1,13 +1,36 @@
 #! /usr/bin/perl
 
+## mdoc2texi.pl -- Convert mdoc tags to texi tags
+##
+## Author:	Harlan Stenn <stenn@ntp.org>
+##		
+##
+##  This file is part of AutoOpts, a companion to AutoGen.
+##  AutoOpts is free software.
+##  AutoOpts is Copyright (C) 1992-2014 by Bruce Korb - all rights reserved
+##
+##  AutoOpts is available under any one of two licenses.  The license
+##  in use must be one of these two and the choice is under the control
+##  of the user of the license.
+##
+##   The GNU Lesser General Public License, version 3 or later
+##      See the files "COPYING.lgplv3" and "COPYING.gplv3"
+##
+##   The Modified Berkeley Software Distribution License
+##      See the file "COPYING.mbsd"
+##
+##  These files have the following sha256 sums:
+##
+##  8584710e9b04216a394078dc156b781d0b47e1729104d666658aecef8ee32e95  COPYING.gplv3
+##  4379e7444a0e2ce2b12dd6f5a52a27a4d02d39d247901d3285c88cf0d37f477b  COPYING.lgplv3
+##  13aa749a5b0a454917a944ed8fffc530b784f5ead522b1aacaf4ec8aa55a6239  COPYING.mbsd
+
 ### To Do:
 
 # the Bl -column command needs work:
 # - support for "-offset" 
 # - support for the header widths
 
-# written by Harlan Stenn
-
 ###
 
 package mdoc2texi;
-- 
2.1.0

